noir-projects/aztec-nr/aztec/src/encrypted_logs/payload.nr

```diff
@@ -76,11 +71,10 @@ pub fn compute_partial_public_log_payload<let P: u32, let M: u32>(
     recipient: AztecAddress,
     sender: AztecAddress,
     plaintext: [u8; P],
-) -> [u8; M] {
+) -> [Field; 1 + (M + 30) / 31] {
```

Comment:
Would copy here the comment from macros so that it's clear why is the return value like that:

"Each field (apart from the tag) contains 31 bytes so the length in fields is computed as ceil(encrypted_log_byte_length / 31) --> we achieve rounding by adding 30 and then dividing without remainder, then add one for the 32 byte tag"

Reply:
Good point, will do!